db/migrate/20201113114332_add_uuid_to_course.rb

```diff
@@ -0,0 +1,13 @@
+class AddUuidToCourse < ActiveRecord::Migration[6.0]
+  def up
+    enable_extension "uuid-ossp" unless extension_enabled?("uuid-ossp")
```

Comment:
@vigneshmsft We'll need this on PAAS. Can it be added?

Reply:
See https://github.com/DFE-Digital/teacher-training-api/pull/1643#discussion_r522865174